addon-test-support/-private/weight-test-modules.js

```diff
@@ -0,0 +1,37 @@
+const WEIGHT_MAP = {jshint:1, eslint:1, unit:10, integration:20, acceptance:150};
+const WEIGHT_DEFAULT = 50;
```

Comment:
Can we call this `DEFAULT_WEIGHT`? It reads more like english that way. I think I mentioned this before in a previous comment.